flink-clients/src/test/java/org/apache/flink/client/deployment/application/ApplicationDispatcherBootstrapITCase.java

```diff
@@ -106,8 +112,16 @@ public void testDispatcherRecoversAfterLosingAndRegainingLeadership() throws Exc
                     JobStatus.RUNNING,
                     deadline);
 
+            // make sure the operator is actually running
+            BlockingJob.awaitRunning(blockId);
+
+            final CompletableFuture<JobResult> firstJobResult =
+                    cluster.requestJobResult(ApplicationDispatcherBootstrap.ZERO_JOB_ID);
             // revoke & re-grant dispatcher leadership
```

Comment:
maybe we should remove this line, because the commands are not connected anymore and the comment add nothing beyond the command itself.

Reply:
👍